### PR TITLE
Update for Command changes

### PR DESCRIPTION
**Pull request by windy1**
SpongeAPI PR: https://github.com/SpongePowered/SpongeAPI/pull/1061

Signed-off-by: Walker Crouse <walkercrouse@hotmail.com>

### CODE REVIEW

File: src/main/java/org/spongepowered/common/command/MinecraftCommandWrapper.java [hunk 1]
```diff
@@ -25,43 +25,22 @@
...
 import static com.google.common.base.Preconditions.checkNotNull;
-
 import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableList;
-import net.minecraft.command.CommandBase;
-import net.minecraft.command.CommandHandler;
-import net.minecraft.command.ICommand;
-import net.minecraft.command.ICommandSender;
-import net.minecraft.command.PlayerSelector;
+import net.minecraft.command.*;
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/command/MinecraftCommandWrapper.java]: Don't do * imports

Comment (windy1) [inline src/main/java/org/spongepowered/common/command/MinecraftCommandWrapper.java]: Yeah sorry it's my damn IDE that keeps sneaking them in. Fixed now.

File: src/main/java/org/spongepowered/common/command/SpongeCommandManager.java [hunk 2]
```diff
@@ -318,6 +316,11 @@ public boolean testPermission(CommandSource source) {
     }
 
     @Override
+    public boolean testSourceType(CommandSource source) {
+        return dispatcher.testSourceType(source);
```
Comment (kashike) [inline src/main/java/org/spongepowered/common/command/SpongeCommandManager.java]: `checkNotNull(source, "source");`
Qualify with `this.`.

File: src/main/java/org/spongepowered/common/command/SpongeCommandManager.java [hunk 3]
```diff
@@ -67,9 +68,6 @@
 import java.util.Set;
 import java.util.function.Function;
 
-import javax.annotation.Nullable;
-import javax.inject.Inject;
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/command/SpongeCommandManager.java]: `javax` should come after `java`, keep these here
Remember - https://docs.spongepowered.org/en/contributing/implementation/codestyle.html

**Pull request closed without merging**